avalon/maya/commands.py

```diff
@@ -18,8 +18,15 @@ def reset_frame_range():
     shot = io.find_one({"name": shot, "type": "asset"})
 
     try:
-        edit_in = shot["data"]["edit_in"]
-        edit_out = shot["data"]["edit_out"]
+
+        frame_start = shot["data"].get(
+            "frameStart",
+            shot["data"].get("edit_in")
```

Comment:
Consider this a question, but should we maybe explicitly comment with these fallbacks that they are due to "Backwards compatibility" - I can imagine newcomers or ourselves in 1-2 years looking at this code and be like "why the hell does it also allow `edit_in`?"

I'd rather have it clear that it's only allowing that fallback due to backwards compatibility.

Reply:
I can really imagine myself in 1-2 weeks looking at this code and be like "who the hell wrote this?" Agree